tasks/tasks.go

```diff
@@ -0,0 +1,27 @@
+package tasks
```

Comment:
I _think_ we can delete these, these are no longer used after the refactor.

Reply:
Yeah, rebasing currently.